spec/features/users/profile_spec.rb

```diff
@@ -1,8 +1,13 @@
 require 'rails_helper'
 RSpec.describe 'user profile page' do
   it 'can show the attributes of the current user', :vcr do
+    allow(BackEndFacade).to receive(:get_user_friends).with(@user.id).and_return([])
+    allow(BackEndFacade).to receive(:get_user_gyms).with(@user.id).and_return([])
+    allow(BackEndFacade).to receive(:get_user_events).with(@user.id).and_return([])
+    allow(BackEndFacade).to receive(:get_user).with(@user.id.to_s).and_return(@user)
```

Comment:
These stubs needed to be added to get the profile spec to pass.

Reply:
Added on my branch, same problem